pkg/ocm/managedcluster_test.go

```diff
@@ -172,6 +172,32 @@ func TestPullManagedCluster(t *testing.T) {
 	}
 }
 
+func TestWithHubAcceptsClient(t *testing.T) {
+	testCases := []struct {
+		accept            bool
+		expectedErrorText string
```

Comment:
Not really, I never expect an error message to be returned from the method so I put it explicitly in the test. Otherwise if a change were made that modified the error in some way this test would never see it.